test/readme/README.template.md

```diff
@@ -188,6 +188,14 @@ needles: ['{a,d}.{b,f}']
 comment: `a.b`, `a.f`, `d.b` and `d.f`
 </example></pre>
 
+### Nested Path
+
+There are two types of nested path matching:
+- `**{...}`: Matches path zero or more times
+- `++{...}`: Matches path one or more times
+
+// todo: fill in details here with examples
```

Comment:
Add notes on string vs array traversal recursively

Reply:
Add note to section "Recursions can be combined with a regex by appending the regex." they can also be combined with groups